sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleCounterSdk.java

```diff
@@ -38,7 +38,9 @@ private DoubleCounterSdk(
 
   @Override
   public void add(double delta, LabelSet labelSet) {
-    bind(labelSet).add(delta);
+    BoundDoubleCounter boundDoubleCounter = bind(labelSet);
+    boundDoubleCounter.add(delta);
+    boundDoubleCounter.unbind();
```

Comment:
Just thinking out loud...what would you think about having `add` return `this` so you could just write:
```
bind(labelSet).add(delta).unbind();
```
in one fluent call?

Reply:
I thought about this, but it is not that useful for users using bounds, but probably doesn't hurt. Happy to accept a followup PR :)

Reply:
yeah, I agree. The bytecode probably ends up looking identical.